cypress/integration/mat.e2e.js

```diff
@@ -0,0 +1,19 @@
+describe('MAT Tests', () => {
+
+  describe('MAT redirections', () => {
+    it('Old /experimental paths redirect to /chart/mat', () => {
+      cy.visit('/experimental/mat?probe_cc=IR&test_name=web_connectivity&category_code=NEWS&since=2022-03-01&until=2022-04-01&axis_x=measurement_start_day&axis_y=probe_asn')
+      cy.url().should('include', '/chart/mat')
```

Comment:
Maybe you can also add checks in here to see if the URL path also includes one of the query parameters?